pydis_site/apps/resources/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from pydis_site.apps.resources import views
+
+app_name = "resources"
+urlpatterns = [
+    path("", views.ResourcesView.as_view(), name="resources"),
```

Comment:
```suggestion
    path("", views.ResourcesView.as_view(), name="index"),
```
I think this `name` might be better suited to `index`? `resources:index` gives me more context than `resources:resources`.